Set up cards

keegan266 opened the pull request.

keegan266 committed:
Set up cards

Trevor266 commented on AngularInventory/src/styles.css:
```diff
@@ -22,9 +22,24 @@ h1 {
+    width: 500px;
+}
+
+.edgeimg {
```
edgeImg doesn't really mean anything, it would be hard to tell how to use this based on the name. Consider renaming it or leaving a comment to explain what it is.

Trevor266 commented on AngularInventory/src/styles.css:
```diff
@@ -22,9 +22,24 @@ h1 {
+    margin-bottom: 100px;
+}
+
+.row {
```
This will overwrite all rows in bootstrap, create a custom row.

Trevor266 commented on AngularInventory/src/styles.css:
```diff
@@ -22,9 +22,24 @@ h1 {
 .card {
-    max-width: 250px;
+    width: 500px;
+    box-shadow: 5px 10px 18px #888888;
```
Use sass to define colors as named variables, and use named variables for colors.

keegan266 committed:
Moved Home Into Component

Moved contents of home page into a component instead of having it in the index

keegan266 committed:
Suggested Changes and new component

Added comment to .edgeimg to better explain what it does and created custom row class for the home page. Created new component for the view orders page

keegan266 committed:
Update app.component.css

keegan266 committed:
Added routing

Added routing to app

keegan266 committed:
Order page table

Created the table and filled with dummy data

keegan266 committed:
Update

Updating Angular

keegan266 committed:
Finished Update

Finished updating Angular to version 12
